src/Raven.Server/Documents/Indexes/Workers/Cleanup/TombstoneIndexItem.cs

```diff
@@ -21,7 +21,7 @@ public DynamicJsonValue ToJson()
             return new DynamicJsonValue
             {
                 [nameof(LowerId)] = LowerId?.ToString(),
-                [nameof(LuceneKey)] = LuceneKey?.ToString(),
+                [nameof(PrefixKey)] = PrefixKey?.ToString(),
```

Comment:
What do we use it for? Do we need to think about backward compatibility here?

Reply:
umm.. I added this class in https://github.com/ravendb/ravendb/pull/18928/files#diff-87af3032e39bc28e6f362a40a3ea798bdd652a52ed796077c8fdf89c78d6938a (for v5.4). we're using it as the key to delete the relevant entries for time series indexes. I don't think backward compatibility is needed here since in v5.4, it's only relevant for Lucene.

Reply:
I just though that maybe we store / serialize this JSON somewhere so we could have issues now during deserialization. If you don't see an issue that's fine.